### DISCUSSION TITLE
AIRSDK.dev Update

### DISCUSSION THREAD

**Discussion by marchbold** [airsdk/Adobe-Runtime-Support#2766] [Show and tell]
Hi all,

Just wanted to give a big shout out to @joshtynjala for converting a lot of the old Adobe *ActionScript 3.0 Developer's Guide* into markdown. 

I've been able to pull a lot of it into the airsdk.dev site and have been through, formatted it, and updated the links so we now have more of the development section fleshed out. 

https://airsdk.dev/docs/development

All this new content is just pulled directly from the old site so if you do have any time to have a read through, make any notes on deprecated / old documentation and let us know we can bring this site up to date.

**Comment by Devin-Licastro** [comment]
Wow that's great, thanks both!

**Comment by raresn** [comment]
Thank you for this  @joshtynjala!

**Comment by joshtynjala** [comment]
I converted the docs to Markdown so that I could adapt some of it for OpenFL (yay, Creative Commons license!). However, I realized that it could also be useful for the AIR SDK website too (especially if Adobe ends up taking their docs down). I'm glad that I could help a bit!

**Comment by 2jfw** [comment]
Awesome! Thanks a lot for this!

**Comment by Michoko92** [comment]
Thank you always for your fantastic job, guys!
